Feat/schedule UI

nam-t24 opened the pull request:
Styling should be done besides the official coloring of the individual event boxes which I believe is being redesigned.

nam-t24 committed:
feat: Set up schedule coloring

nam-t24 committed:
chore: Fix merge conflicts with develop

nam-t24 committed:
Merge branch 'develop' of https://github.com/acmutd/hackutd-ix-site into feat/ScheduleUI

nam-t24 committed:
test: Test css classes for schedule

NickBurnett commented:
@nam-t24 Try changing the ".css-4plepo .MainLayout-stickyElement" background property through the DOM and see if you can get it to work through the component after.

abdullah50053 committed:
Merge branch 'develop' into feat/ScheduleUI

nam-t24 committed:
feat: Finish styling for main calendar table

abdullah50053 committed:
Merge pull request #53 from acmutd/develop

chore: Update scheduleUI branch

nam-t24 committed:
refactor: Change schedule css classes to more gneralized classes

More generalized classes would work on all calendar instances instead of
specific instances

nam-t24 committed:
fix: Fix tracks css classes that got deleted

nam-t24 committed:
feat: Finish schedule page styling

abdullah50053 commented:
Probably going to change the card colors but otherwise looks good. Waiting to get the schedule from experience before merge.

nam-t24 committed:
refactor: Change event colors and extend schedule hours
